### PR TITLE
Shorten Softmx test name for it work work on Windows

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/215
Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Tested Here : https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/712/console

File: systemtest/playlist.xml [hunk 1]
```diff
@@ -671,7 +671,7 @@
 	</test>
 	
 	<test>
-		<testCaseName>SharedClassesWorkloadTest_Softmx_IncreaseDecrease</testCaseName>
+		<testCaseName>SharedClassesWorkload_Softmx_ID</testCaseName>
```
Comment (smlambert) [inline systemtest/playlist.xml]: Can this be changed to Softmx_UpDown or something that is not mistakenly read as ID = identity ?

**Comment by smlambert** [comment]
SharedClasses is often reduced to SC in other places... I am not a big fan of that, but mention it in the event its already shortened that way in other system tests.

File: systemtest/playlist.xml (hunk 1 above)
Comment (Mesbah-Alam) [reply systemtest/playlist.xml]: Good suggestion. Updated!

**Comment by llxia** [comment]
- Since Grinder in Adopt works anyway, we may want to test in Openj9 to make sure this PR works.
- Please backport this PR into the extraVariations branch. Thanks

**Comment by llxia** [comment]
> Please backport this PR into the extraVariations branch. Thanks

Thanks for Shelley suggestion to use `special` level for extra variations. We do not backport PR into the extraVariations branch. https://github.com/AdoptOpenJDK/openjdk-tests/issues/775

I will add extra variations into master as `special` level.

**Comment by Mesbah-Alam** [comment]
Tested in Grinder at OpenJ9 : https://ci.eclipse.org/openj9/view/Test/job/Test-Grinder/132/console 

The new names should work in the regular builds as well, since the new names have been made shorter than `SharedClassesWorkloadTest_Softmx_Increase` - which worked successfully on the same machine where the others failed: https://ci.eclipse.org/openj9/view/Test/job/Test-extended.system-JDK11-win_x86-64_cmprssptrs/120/tapResults/

**Pull request merged**